features/brokers/broker_create_hc4cc_quote.feature

```diff
@@ -43,4 +43,24 @@ Feature: Broker HC4CC quote creation
     Then Primary broker should see plan names in employee costs
     Then Primary broker should see employee costs download pdf button
     Then Primary broker should see total HC4CC subcidy applied amount
-    And Primary broker publishes the quote and sees successful message of published quote
\ No newline at end of file
+    And Primary broker publishes the quote and sees successful message of published quote
+
+  Scenario: Broker should be able to create a copy of a quote with OSSE eligibility
+    Given prospect employer exist for District Brokers Inc
+    And Primary broker clicks Actions dropdown and clicks Create Quote
+    And Primary Broker enters quote name
+    And Primary Broker should see HC4CC option
+    Then Primary Broker selects quote as HC4CCC quote
+    And Primary broker clicks on Select Health Benefits button
+    And Primary broker should see metal level non bronze options
```

Comment:
lines `49-55` can you move these to `background`  to avoid duplicating them in scenarios